<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="rline">
    <property name="geometry">
     <rect>
      <x>603</x>
      <y>-40</y>
      <width>20</width>
      <height>561</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>41</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Clear Canvas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Romb</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rectangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Polygon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Star</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="shape">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Shape</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawB">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enable drawing mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="area">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Area:</string>
    </property>
   </widget>
   <widget class="Line" name="lline">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>20</width>
      <height>471</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>481</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="perim">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Perimeter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="options">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QLabel" name="areaVal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="perimVal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="centLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Center:</string>
    </property>
   </widget>
   <widget class="QLabel" name="centX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="centY">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>91</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="n_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Edges:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="n">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="aRad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>OuterRad</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="radB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="bRad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>InnerRad</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="radA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>91</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Length:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="wid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="hei">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>40</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>radA:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="FRad">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="SRad">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>radB:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>91</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>First</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Second</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Third</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="first">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="second">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="third">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>91</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Diam1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Diam2</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="diam1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="diam2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="done">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Done drawing</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="center">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Center:</string>
     </property>
    </widget>
    <widget class="QLabel" name="x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="x_inp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="y">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="y_inp">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="rotate">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rotate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="leftB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rightB">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QPushButton" name="plusB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minusB">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="scale">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Edges:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="edgs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
